src/Symfony/Component/VarDumper/Dumper/ContextProvider/RequestContextProvider.php

```diff
@@ -45,7 +45,7 @@ public function getContext(): ?array
             'uri' => $request->getUri(),
             'method' => $request->getMethod(),
             'controller' => $controller ? $this->cloner->cloneVar($controller) : $controller,
-            'identifier' => spl_object_hash($request),
+            'identifier' => hash('crc32b',  spl_object_hash($request).$_SERVER['REQUEST_TIME_FLOAT']),
```

Comment:
`xxh128` is already used elsewhere in Symfony for its excellent performance, low collision risk, and suitability for non-cryptographic identifiers.
Can you switch to xxh128 here?

Reply:
that'd make sense to me. Also we could use spl_object_id